Gemfile

```diff
@@ -4,7 +4,7 @@ source 'http://rubygems.org'
 
 gemspec
 
-group :development do
+group :development, :test do
```

Comment:
it adds always 0.51 → maybe you have to remove the line in gem spec?

Reply:
>it adds always 0.51

Yes, How about https://github.com/ruby-grape/grape-entity/pull/278/commits/b66105fb2b7e52df28d369d1f1cd1553fcf6ff48?